show/feature.py

```diff
@@ -0,0 +1,56 @@
+import click
+from natsort import natsorted
+from tabulate import tabulate
+
+from utilities_common.cli import pass_db
+
+#
+# 'feature' group (show feature ...)
+#
+@click.group(name='feature', invoke_without_command=False)
```

Comment:
Add `cls=AbbreviationGroup`

Reply:
how to test the abbreviations?

Reply:
Type enough chars to distinguish the command from other commands and the subcommand from other subcommands. E.g., `show fea au`, `show fea st`, etc.

Reply:
```
admin@vlab-01:~$ show feat st
Feature         State     AutoRestart
--------------  --------  -------------
bgp             enabled   enabled
database        enabled   disabled
dhcp_relay      enabled   enabled
lldp            enabled   enabled
mgmt-framework  enabled   enabled
nat             disabled  enabled
pmon            enabled   enabled
radv            enabled   enabled
sflow           disabled  enabled
snmp            enabled   enabled
swss            enabled   enabled
syncd           enabled   enabled
teamd           enabled   enabled
telemetry       enabled   enabled
```